addons/modules/fnc_getStringPos.sqf

```diff
@@ -0,0 +1,47 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getStringPos
+
+Description:
+    A function used to get the position of a string with a passed typeName.
+
+Parameters:
+    - Number (Number from 0 to 5)
+        0 - Return 0 to decide what to do later in script
+        1 - Object or location
+        2 - Group
+        3 - Array
+        4 - Marker
+        5 - Task
+    - String (XYZ, Object, Location, task, group, or Marker)
+
+Example:
+    (begin example)
+    [1, "player"] call CBA_fnc_getStringPos;
+    (end)
+
+Returns:
+    Position [0,0,0] or 0
```

Comment:
This says it could return an array with values [0,0,0] or just a 0. 

Could probably add when it returns what (invalid input results in a 0, I assume?).